modules/tinymce/src/themes/silver/main/ts/ui/dialog/Collection.ts

```diff
@@ -62,7 +58,7 @@ export const renderCollection = (spec: CollectionSpec, providersBackstage: UiFac
       const ariaLabel = itemText.replace(/\_| \- |\-/g, (match) => mapItemName[match]);
 
       const readonlyClass = providersBackstage.isReadOnly() ? ' tox-collection__item--state-disabled' : '';
-      return `<div class="tox-collection__item${readonlyClass}" tabindex="-1" data-collection-item-value="${escapeAttribute(item.value)}" title="${ariaLabel}" aria-label="${ariaLabel}">${iconContent}${textContent}</div>`;
+      return `<div class="tox-collection__item${readonlyClass}" tabindex="-1" data-collection-item-value="${Entities.encodeAllRaw(item.value)}" title="${ariaLabel}" aria-label="${ariaLabel}">${iconContent}${textContent}</div>`;
```

Comment:
> This function will produce larger text contents since it doesn't know if the context is within a attribute or text node.
That's from a comment above `encodeAllRaw` (I hadn't seen it before so I looked it up) — is there an alternative that would be more appropriate here as we know the string is meant to be within an attribute?

Reply:
I had a look and couldn't find any :disappointed: We could split the string and use `encodeRaw` or another option would be to add an additional argument to `encodeAllRaw` so we could provide that hint maybe?